flatpaks/Makefile

```
@@ -40,4 +41,8 @@ clean:
 	$(if $(wildcard repo),rm -Rf repo)
 	$(if $(wildcard list.txt),rm list.txt)
 
-.ONESHELL:
\ No newline at end of file
+container-%:
```

checkmake (0.3.2)

[warning] Target "container-%" should be declared PHONY.

(phonydeclared)